draft-looker-cfrg-bbs-signatures.md

```diff
@@ -1259,6 +1271,156 @@ Along with the SK value as defined in (#key-pair) as inputs into the Sign operat
 {{ $signatureFixtures.signature004.signature }}
 ```
 
+### Valid Single Message Proof
+
+Using the following signature
+
+```
+{{ $proofFixtures.proof001.signature }}
+```
+
+Which signed the following message
+
+```
+{{ $proofFixtures.proof001.messages[0] }}
+```
+
+And header
+
+```
+{{ $proofFixtures.proof001.header }}
+```
+
+And generating a proof revealing the only message signed in the generated proof with the following presentation header
+
+```
+{{ $proofFixtures.proof001.presentationMessage }}
+```
+
+And public key of the signer
+
+```
+{{ $proofFixtures.proof001.signerPublicKey }}
+```
+
+Yields the following proof
+
+```
+{{ $proofFixtures.proof001.proof }}
+```
+
+### Valid Multi-Message Proof All Messages Revealed
+
+Using the following signature
+
+```
+{{ $proofFixtures.proof002.signature }}
+```
+
+Which signed the following messages
+
+```
+{{ $proofFixtures.proof002.messages[0] }}
+
+{{ $proofFixtures.proof002.messages[1] }}
+
+{{ $proofFixtures.proof002.messages[2] }}
+
+{{ $proofFixtures.proof002.messages[3] }}
+
+{{ $proofFixtures.proof002.messages[4] }}
+
+{{ $proofFixtures.proof002.messages[5] }}
+
+{{ $proofFixtures.proof002.messages[6] }}
+
+{{ $proofFixtures.proof002.messages[7] }}
+
+{{ $proofFixtures.proof002.messages[8] }}
+
+{{ $proofFixtures.proof002.messages[9] }}
+```
+
+And header
+
+```
+{{ $proofFixtures.proof002.header }}
+```
+
+And generating a proof revealing all the message signed in the generated proof with the following presentation header
+
+```
+{{ $proofFixtures.proof002.presentationMessage }}
+```
+
+And public key of the signer
+
+```
+{{ $proofFixtures.proof002.signerPublicKey }}
+```
+
+Yields the following proof
```

Comment:
Maybe it's naive, but could this be confusing until we have the ability to test with a deterministic proofGen??